h/services/user_signup.py

```diff
@@ -115,10 +107,6 @@ def signup(self, require_activation=True, **kwargs):
         sub = Subscriptions(uri=user.userid, type="reply", active=True)
         self.session.add(sub)
 
-        # Record a registration with the stats service
-        if self.stats is not None:
-            self.stats.incr("auth.local.register")
```

Comment:
`UserSignupService` was [added when we wanted to add API support for signing-up new users](https://github.com/hypothesis/h/pull/3706) and, because the user signup page incremented this `"auth.local.register"` stat, so did the `UserSignupService` from its conception